scripts/lib/CIME/XML/grids.py

```diff
@@ -57,6 +57,8 @@ def get_grid_info(self, name, compset, driver):
         atmnlev = None
         lndnlev = None
 
+        print("compset is now {} ".format(compset))
```

Comment:
Just checking: did you mean to leave this in here, or should this be changed to something like a logging.debug?